packages/help-center/src/components/help-center-gpt.tsx

```diff
@@ -69,26 +69,16 @@ export function HelpCenterGPT() {
 
 	const query = message ?? '';
 
-	// First fetch the links
-	const { data: links, isError: isLinksError } = useJetpackSearchAIQuery( {
-		siteId: '9619154',
-		query: query,
-		stopAt: 'urls',
-		enabled: true,
-	} );
-
 	// Then fetch the response
```

Comment:
Suuuper nitpick, but we can remove this comment now too, since the `// First fetch the links` above was what gave it context and meaning.